release-notes/topics/aap-22.adoc

```diff
@@ -0,0 +1,15 @@
+[[aap-2.2-intro]]
+= Ansible Automation Platform 2.2
+
+Red Hat Ansible Automation Platform simplifies the development and operation of automation workloads for managing enterprise application infrastructure lifecycles. It works across multiple IT domains including operations, networking, security, and development, as well as across diverse hybrid environments. Simple to adopt, use, and understand, Red Hat Ansible Automation Platform provides the tools needed to rapidly implement enterprise-wide automation, no matter where you are in your automation journey.
+
+.Enhancements
+
+* Added the Automation Services Catalog as a component for the Ansible Automation Platform
```

Comment:
I'd like to qualify this a bit.  Add "on-prem" preceding component. 

So it will read, "Added the Automation Services Catalaog as an on-prem component..."

ASC has always been a component of AAP.  It's just been in console.redhat.com and is now sunset from there as on-prem only.

Reply:
Since we mention private services catalog below, might want to use the same wording instead?

Reply:
These changes were added but I'm not sure I understand what change you were requesting with the above statement. "Since we mention private services catalog below, might want to use the same wording instead?" Can you clarify, or is it ok with the other fix I added?